src/Logger/StepupRequestIdSariLogger.php

```diff
@@ -41,6 +44,9 @@ public function __construct(
         $this->sariLogger = $sariLogger;
     }
 
+    /**
```

Comment:
With early return, you can skip the `else` and the warning suppression

Reply:
Good point.